runbotics/runbotics-orchestrator-ui/src/src-landing/views/sections/BenefitsSection/BenefitsCarusel/BenefitsCarusel.types.ts

```diff
@@ -0,0 +1,8 @@
+import { TranslationsDescriptors } from '#src-app/translations/translations';
+
+export interface Slide {
+    titleKey: keyof TranslationsDescriptors;
```

Comment:
imo better use specific descriptors provided for this section

Reply:
dont know what's the use case but sure